static/js/theme.js

```diff
@@ -0,0 +1,34 @@
+const lightThemeCSS = "/static/css/foundation-light.min.css";
+const darkThemeCSS = "/static/css/foundation-dark.min.css";
+var toDark, toLight;
```

Comment:
These variables are only used in toggleTheme, why pollute the global namespace?

Reply:
Okay Sir, should ```toLight``` & ```toDark``` remain global variables? ```toLight``` is being used in document.ready() also...